packages/client/_src/Resources/Blog/PublishPost.ts

```diff
@@ -0,0 +1,10 @@
+import { BlogPostId } from "@effect-ts-app/boilerplate-types/Blog"
+import { OperationId } from "../../Views.js"
+
+@allowAnonymous
+@allowRoles("user")
+export class PublishPostRequest extends Post("/blog/posts/:id/publish")<PublishPostRequest>()({
+  id: prop(BlogPostId)
+}) {}
+
+export const PublishPostResponse = OperationId
```

Comment:
What is the role of operation ids?

It seems to me that `OperationId` is sort of an internal identifier for Effects that do multiple things.

Since when we publish we publish N things, I will have N operation identifiers, is my intuition correct?

Reply:
The OperationId is used to track long running Operations.
https://github.com/effect-ts-app/boilerplate/pull/6/commits/caa7caf6bbb6df417d34bd40df48105f1f899a45